src/core/LanguageMenu/LanguageMenu.md

```diff
@@ -0,0 +1,28 @@
+<!-- ```js
+import { LanguageMenu, LanguageMenuItem, LanguageMenuLink } from 'suomifi-ui-components';
+
+<LanguageMenu className="language-menu-test" name="LanguageMenu">
+  <LanguageMenuItem onSelect={() => console.log('LanguageMenu test 1')}>
+    Item 1
+  </LanguageMenuItem>
+  <LanguageMenuItem onSelect={() => console.log('LanguageMenu test 2')}>
+    Item 2
+  </LanguageMenuItem>
+  <LanguageMenuLink href="http://www.suomi.fi/">Suomi.fi</LanguageMenuLink>
+</LanguageMenu>;
+``` -->
+
+```js
+import {
+  LanguageMenu,
+  LanguageMenuItem,
+  LanguageMenuLink
+} from 'suomifi-ui-components';
+
+<LanguageMenu className="language-menu-language-test" name="FI">
+  <LanguageMenuItem onSelect={() => console.log('FI')} selected>
+    Suomeksi (FI)
+  </LanguageMenuItem>
+  <LanguageMenuLink href="/sv">På svenska (SV)</LanguageMenuLink>
```

Comment:
Would it be clear enough without the href, since the link actually works but doesn't lead anywhere relevant?

Reply:
let's leave it for now. It demonstrates the use well and it should be quite clear to our demographic that links in the demo are not meant to work.